models/ens/ens_resolver_addresses.sql

```diff
@@ -115,7 +115,7 @@ WHERE latest_ens = 1
         ) names
 )
 
-SELECT address
+SELECT distinct address
```

Comment:
`distinct` is probably needed because Registrar.setName and Resolver.setName are redundant ( Registrar.setName usually calls Resolver.setName internally like https://github.com/ensdomains/ens-archived-contracts/blob/main/contracts/ens/ReverseRegistrar.sol#L81